spring-cloud-starter-stream-source-twitterstream/src/main/java/org/springframework/cloud/stream/app/twitterstream/source/TwitterStreamMessageProducer.java

```diff
@@ -16,55 +16,68 @@
 
 package org.springframework.cloud.stream.app.twitterstream.source;
 
-import java.net.URI;
-
 import org.springframework.social.support.URIBuilder;
 import org.springframework.social.twitter.api.impl.TwitterTemplate;
 import org.springframework.util.StringUtils;
 
+import java.net.URI;
+
 /**
- *  {@link org.springframework.integration.core.MessageProducer} implementation to send Twitter stream messages.
+ * {@link org.springframework.integration.core.MessageProducer} implementation to send Twitter stream messages.
  *
  * @author Ilayaperumal Gopinathan
  * @author Gary Russell
  */
 class TwitterStreamMessageProducer extends AbstractTwitterInboundChannelAdapter {
 
-	private static final String API_URL_BASE = "https://stream.twitter.com/1.1/statuses/";
+    private static final String API_URL_BASE = "https://stream.twitter.com/1.1/statuses/";
```

Comment:
The indents must be `tabs`, not `spaces`, please.
Otherwise all the class looks like your changes.